test/token/ERC20/ERC20.test.js

```diff
@@ -287,6 +287,46 @@ contract('ERC20', function (accounts) {
     });
   });
 
+  describe('_update', function () {
+    const amount = 1;
+    
+    it('from is the zero address', async function () {
```

Comment:
Let's add an assertion for the transfer event in this test.

Reply:
Fixed in last commit.